waku/standards/core/17/rln-relay.md

```diff
@@ -6,11 +6,12 @@ status: draft
 tags: waku-core
 editor: Sanaz Taheri <sanaz@status.im>
 contributors:
-  - Oskar Thorén <oskarth@titanproxy.com>
```

Comment:
I think Oskar wishes this email is used, not the status.im one

Reply:
@rymnc some of the pr being transferred from the old repo will not be updated. I have not looked over any of these, before merging everything should be checked.